### PR TITLE
feat(jobs): add schedule analysis page

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Description

Added schedule analysis page

### Changes

- Added schedule analysis page
- Updated browse api to use `query-string` library to sanitize url query

## Screenshot
image

## How to test

Go to `http://localhost:3000/jobs/scheduleAgents`

### CODE REVIEW

File: src/pages/Jobs/ScheduleAgents/index.js [hunk 1]
```diff
@@ -16,18 +16,231 @@
...
+      name: "Software Repository",
+      description: "Top Folder",
+      parent: null,
+    },
+  ];
+  const initialBrowseData = {
+    page: 1,
+    limit: 100,
+    groupName: "",
+    recursive: true,
```
Comment (GMishx) [inline src/pages/Jobs/ScheduleAgents/index.js:71]: Please set recursive to false. We only need data from current folder not every folder.

File: src/pages/Jobs/ScheduleAgents/index.js [hunk 2]
```diff
@@ -16,18 +16,231 @@
...
+import { getBrowseData } from "../../../services/browse";
 
 const ScheduleAgents = () => {
+  const initialScheduleAnalysisData = {
+    folderId: 1,
+    uploadId: 1,
+  };
+  const initialScanFileData = {
+    analysis: {
+      bucket: true,
```
Comment (GMishx) [inline src/pages/Jobs/ScheduleAgents/index.js:36]: Please read buckets as well from UI.

Comment (Aman-Codes) [reply src/pages/Jobs/ScheduleAgents/index.js:36]: @GMishx Updated it.

File: src/pages/Jobs/ScheduleAgents/index.js (hunk 1 above)
Comment (Aman-Codes) [reply src/pages/Jobs/ScheduleAgents/index.js]: Changed recursive to false

File: src/pages/Jobs/ScheduleAgents/index.js (hunk 2 above)
Comment (GMishx) [reply src/pages/Jobs/ScheduleAgents/index.js:36]: Now bucket is checked by default. Also, in current UI it is the first item, please shift accordingly.

Comment (Aman-Codes) [reply src/pages/Jobs/ScheduleAgents/index.js]: @GMishx Updated it

**Review by GMishx — APPROVED** [review]

**Pull request merged**